test/integration/capacity.test.ts

```diff
@@ -192,6 +192,7 @@ describe('capacity', () => {
     })
 
     it('non-existent Capacity ID should return nothing - 404', async () => {
+      await cleanup()
```

Comment:
what's the purpose of the `cleanup` here?

Reply:
To empty the db and check it returns nothing.

Reply:
to reset the database so we can assert for getAll if no records are preset, this has been slightly updated

Reply:
the test is a `404` because there's no capacity with ID `nonExistentId`, there should be no need to reset the db since no seed adds a record for `nonExistentId`

Reply:
yea, sorry this is my bad, took a while to understand what was done.